### DISCUSSION TITLE
Transport endpoint is not connected Termux

### DISCUSSION THREAD

**Discussion by sirusdas** [termux/termux-app#2504] [Q&A] [answered]
I want to run a python file using cron with termux-chroot enabled which i am not getting any success, and while executing it i am getting an error known as "Transport endpoint is not connected".

As per my current findings A file executes when placed at /data/data/com.termux/files/home/storage dir

but a file wont execute when placed in /data/data/com.termux/files/home/storage/shared/Termux dir

**Comment by sylirre** [comment, marked as answer]
The storage dir is still part of Termux home. The execute permission as well as access modes (chmod) have effect here. However if you change directory to `~/storage/shared/Termux` or any other directory under `~/storage/shared` the execute permissions would be lost. The Android shared storage is mounted with noexec option.

See storage feature comparison table at https://wiki.termux.com/wiki/Internal_and_external_storage
